deploy/crds/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -96,6 +96,12 @@ spec:
                             enum:
                               - ephemeral
                               - persistent
+                          retention:
+                            description: Time duration Prometheus shall retain data for. Default
+                              is '24h', and must match the regular expression `[0-9]+(ms|s|m|h|d|w|y)`
+                              (milliseconds seconds minutes hours days weeks years).
```

Comment:
Never seen retention times in milliseconds before, but sure! Lol

Reply:
lol yea, but it's valid based on what can be passed to the Prometheus Operator. I just copy/pasted from the Prometheus CRD :D